### ISSUE TITLE
Update tensorflow version

### ISSUE THREAD

**Issue by idofr** [dwiel/tensorflow_hmm#11]
Thanks for sharing 👍 , however, quite a lot of tf function calls here are deprecated.

**Comment by dwiel** [comment]
Yeah, I haven't been using this lately so it hasn't been kept up to date. I'd definitely welcome a pull request to bring it up to date.
